mgmt/RecordsConfig.cc

```diff
@@ -1212,6 +1212,8 @@ static const RecordElement RecordsConfig[] =
   //##############################################################################
   {RECT_CONFIG, "proxy.config.ssl.enabled", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.ssl.session.ticket.enabled", RECD_INT, "1", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
Also, I'm a bit torn on what the correct nomenclature should be. We have ``proxy.config.ssl.server.ticket_key.filename`` so maybe this should be ``proxy.config.ssl.server.session_ticket.enable``? But there are also quite a few SSL settings that don't include the ``server`` component (even though they are server-specific).

I believe that the original intent was that ``proxy.config.ssl.foo`` would apply ``foo`` to both client and server sessions. This is the way the SSL context options work (see ``proxy.config.ssl.compression`` for example).